packages/health-check/src/checks/critical.js

```diff
@@ -108,7 +108,7 @@ async function genericAccessCheck(name, url, done) {
 
 const checks = [uploadCheck, websiteCheck, downloadCheck, skylinkSubdomainCheck, handshakeSubdomainCheck];
 
-if (process.env.ACCOUNTS_ENABLED) {
+if (process.env.ACCOUNTS_ENABLED === "1") {
```

Comment:
I personally don't like using 1 and 0 for 'true' and 'false'. In an `.env` file I guess you'd usually write:
`process.env.ACCOUNTS_ENABLED=true` rather than `process.env.ACCOUNTS_ENABLED=1`.